examples/clinical_pipeline/README.md

```diff
@@ -18,7 +18,7 @@ named entities corresponding to diseases, symptoms and related concepts.
 Before running the pipeline, we need to download the models
 
 ```bash
-python ../../fortex/huggingface/biobert_ner/download_models.py 
+python ./biober_ner/download_models.py 
```

Comment:
I don't think the `biober_ner` folder should be included, maybe just the download script.